Cargo.toml

```diff
@@ -63,3 +63,6 @@ legacy_compatibility = []
 u64_backend = ["curve25519-dalek/u64_backend"]
 u32_backend = ["curve25519-dalek/u32_backend"]
 simd_backend = ["curve25519-dalek/simd_backend"]
+
+[patch.crates-io]
+curve25519-dalek = { git = "https://github.com/mobilecoinfoundation/curve25519-dalek.git", rev = "8791722e0273762552c9a056eaccb7df6baf44d7" }
```

Comment:
This can be removed once https://github.com/dalek-cryptography/ed25519-dalek/pull/193 is published